### ISSUE TITLE
Inconsistent episodes on Taiga vs MAL

### ISSUE THREAD

**Issue by paulo27ms** [erengy/taiga#870]
Version: 1.4.0-alpha
For some reason Taiga is saying I have 11,520 episodes watched, while MAL says 11,506 watched. This isn't a sync problem as watching 1 episode increases in both.
I have deleted my user data from Taiga and even exported my user data from MAL and Taiga and cleaned it up and compared and everything is the exact same (anime rewatched and episodes for all anime).
Not sure what could be causing the issue but can provide whatever is needed.

I thought at first that it was an issue with how like 3 years ago I rewatched a season of Haruhi (which has 14 episodes) and Taiga registered the rewatch, but deleting my user data didn't help and I reckon Taiga probably stopped using that since it can get the rewatched value from MAL's API.

EDIT: The inconsistency, as indicative by my watch time on Taiga vs MAL seems to be around 634 minutes. Which would be ~45 minutes per episode.

There's also no anime that's currently being rewatched.

**Comment by erengy** [comment]
I looked into this but couldn't find anything odd with your list (e.g. progress values exceeding total episode count).

Not sure how MAL counts things, but Taiga's calculation is quite simple:

https://github.com/erengy/taiga/blob/146c2c676fd768b4a8dfc76b207b3fd04a46c3c3/src/taiga/stats.cpp#L54-L67

You could do something like this, though it'd take a while:

1. Import your anime list to a test account and make sure the inconsistency is still there.
2. Delete some parts of your list until the episode difference changes.
3. Clear your list and import only the last deleted part.
4. Repeat steps 2-3 until you figure out which anime makes the difference.

**Comment by paulo27ms** [comment]
Actually I was analyzing my list and just figured the issue out by looking at my rewatches and their progress.
I have Ajin marked as rewatched twice but it's currently on episode 6 (not marked as rewatching).
This means that MAL is doing 6*3 instead of 13*2+6, so 18 vs 32, which is a 14 episode difference.
I was also doing Progress*(Rewatches+1) since I also assumed no rewatches in progress, which returned the same value for MAL and Taiga for my list.
So it's a MAL bug and Taiga is the one taking the correct approach to episode counting.
The issue was fixed when I marked it as completed.
